tests/kubernetes/kubernetes_test.go

```diff
@@ -0,0 +1,58 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/ashraffouda/grid-provider/tests"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+func TestKubernetesDeployment(t *testing.T) {
+	/* Test case for deployeng a k8s.
+
+	   **Test Scenario**
+
+	   - Deploy a k8s.
+	   - Check that the outputs not empty.
+	   - Up wireguard.
+	   - Check that master is reachable
```

Comment:
this is not enough

need to do more
- reachability of everything
- network configurations
- all in ready state
- ensure that everything getting exactly the specs requested in the provisioning